src/components/menu/LoadModule.js

```diff
@@ -331,6 +375,7 @@ class LoadModule extends Component {
                          {Object.keys(this.props.modules).length > 0 ? <li className={(this.state.selectedOption === 'my') ? 'selected' : ''}><button className='btn btn-link' onClick={this.onOptionClick('my')}>My Modules</button></li> : ''}
```

Comment:
Another possible enhancement would be to not show the Local Storage tab unless there are any modules in there. See this "my modules" for an example

Reply:
So I tried to do this with the same kind of Object.keys implementation as "My Modules." The main difference is that the My Modules code checks on the `modules` prop, whereas the local storage stuff initializes a state variable when the component updates to have local storage selected.

To implement this, I think we'd probably have to move the logic for initializing the local storage module back out to the Editor component rather than load modules? There could be another way, but unfortunately it doesn't seem as simple. Would you be willing to compromise and have me just render a message that says that none exist in local storage by default ;)